src/components/stories/inputField.stories.ts

```
@@ -0,0 +1,51 @@
+  },
+}
+
+export const ReadOnly: Story = {
+  args: {
+    defaultValue: 'read only',
+    readonly: true,
+    size: 'lg',
+  },
+}
+
+export const Number: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)